.circleci/config.yml

```diff
@@ -69,45 +74,14 @@ workflows:
   ci:
     jobs:
       - test:
-          name: 'Rails 5.1 - Ruby 2.4'
-          version: '2.4'
-          rails-version: '5.1'
-          sequel-version: '5.0'
-      - test:
-          name: 'Rails 5.1 - Ruby 2.5'
-          version: '2.5'
-          rails-version: '5.1'
-          sequel-version: '5.0'
-      - test:
-          name: 'Rails 5.1 - Ruby 2.6'
-          version: '2.6'
-          rails-version: '5.1'
-          sequel-version: '5.0'
-
-
-      - test:
-          name: 'Rails 6.0 - Ruby 2.5'
-          version: '2.5'
-          rails-version: '6.0'
-          sequel-version: '5.0'
-      - test:
-          name: 'Rails 6.0 - Ruby 2.6'
-          version: '2.6'
-          rails-version: '6.0'
-          sequel-version: '5.0'
-      - test:
-          name: 'Rails 6.0 - Ruby 2.7'
-          version: '2.7'
-          rails-version: '6.0'
-          sequel-version: '5.0'
-
-      - test:
-          name: 'Rails 6.1 - Ruby 2.7'
-          version: '2.7'
-          rails-version: '6.1'
-          sequel-version: '5.0'
+          matrix:
+            parameters:
+              version: [ '2.5', '2.6', '2.7', '3.0', '3.1' ]
+              rails-version: [ '6.0', '6.1' ]
+              sequel-version: [ '5.0' ]
       - test:
-          name: 'Rails 6.1 - Ruby 3.0'
-          version: '3.0'
-          rails-version: '6.1'
-          sequel-version: '5.0'
+          matrix:
+            parameters:
+              version: ['3.0', '3.1']
+              rails-version: ['7.0']
+              sequel-version: ['5.0']
```

Comment:
first time I'm seeing it written like it's pronounced

Reply:
Haha, yes this is for a gem named `sequel` 😄